src/tools/rest-frontend/resources/grunt-tasks/grunt-create-website-news.js

```diff
@@ -57,7 +57,6 @@ module.exports = function (grunt) {
                     shortDesc = shortDesc[1];
                 } else {
                     shortDesc = '';
-                    grunt.log.warn('News post ' + post.name + ' has no description matching the configured regex');
```

Comment:
It is no log but basically stdout/stderr. I re-add the warnings... we should be able to get rid of this one if we add `- shortDesc: ` without content to news if they dont have a short description (which doesnt make sense, but ok).